src/analysis/typepal/StringSimilarity.rsc

```diff
@@ -0,0 +1,59 @@
+module analysis::typepal::StringSimilarity
+
+import List;
+import String;
+import analysis::typepal::TModel;
+
+@synopsis{Tryadic minimum function on integers}
+int min(int a, int b, int c)
+=   a < b ? (a < c ? a : c) : (b < c ? b : c);
+
+@synopsis{Calculate the Levenshtein distance of 2 strings}
+int lev(str a, str b){
+    int sizea = size(a);
+    int sizeb = size(b);
+
+    @memo
```

Comment:
Should we configure the memo to timeout after a few minutes?

Reply:
Yes will do